packages/framework/src/resources/workflow.ts

```diff
@@ -376,3 +382,38 @@ function prettyPrintDiscovery(discoveredWorkflow: DiscoverWorkflowOutput): void
     });
   });
 }
+
+function transformPreference(preference?: WorkflowOptionsPreference): DiscoverWorkflowOutputPreference {
```

Comment:
@rifont what do you think about export these help functions to another file/files?

Reply:
Maybe by creating a workflow folder and then make this file index.ts? And move all help functions to separate files?

Reply:
So it would be 8 files instead of one to reduce the length of this one.

Reply:
I like the suggestion, it will also promote stronger unit test coverage by nature of having smaller files that are more easily testable.